packages/sitecore-jss/src/feaas/themes.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  ComponentRendering,
+  HtmlElementRendering,
+  LayoutServiceData,
+  RouteData,
+  getFieldValue,
+} from '../layout';
+import { HTMLLink } from '../models';
+
+/**
+ * Pattern for library ids
+ * @example -library--foo
+ */
+const FEAAS_LIBRARY_ID_REGEX = /-library--([^\s]+)/;
+
+export const FEAAS_SERVER_URL = 'https://feaas.blob.core.windows.net';
+
+/**
+ * Walks through rendering tree and returns list of links of all FEAAS Component Library Stylesheets that are used
+ * @param {LayoutServiceData} layoutData Layout service data
+ * @param {string} [serverUrl] server URL, default is @see {FEAAS_SERVER_URL} url
+ * @returns {HTMLLink[]} library stylesheet links
+ */
+export function getFEAASLibraryStylesheetLinks(
+  layoutData: LayoutServiceData,
+  serverUrl?: string
+): HTMLLink[] {
+  const ids = new Set<string>();
+
+  if (!layoutData.sitecore.route) return [];
+
+  traverseComponent(layoutData.sitecore.route, ids);
+  [...ids].forEach((id) => {
+    console.log(getStylesheetUrl(id, serverUrl));
+  });
```

Comment:
This bit of code looks like extraneous debug code. Forget to remove?

Reply:
yep, true. thanks for the heads up!